scapy/config.py

```diff
@@ -875,6 +875,7 @@ class Conf(ConfClass):
         'smbclient',
         'smbserver',
         'snmp',
+        'stamp',
```

Comment:
This sounds like a rather uncommon layer. Don't you think it should rather go in `contribs/` ? (see [this](https://scapy.readthedocs.io/en/latest/build_dissect.html#add-new-protocols-to-scapy))

Reply:
I was not aware of the difference between `contrib` and `layers`. I agree that STAMP should go in the `contrib/`. I updated the PR to move stamps to `contrib/`